float32[] grasp_config
string prior_name
int32[] sparse_voxel_grid
int32[] voxel_grid_dim
float32[] object_size
---
float32[] posterior_grad
float32[] likelihood_grad
float32[] prior_grad